exec/slicemachine.go

```diff
@@ -600,7 +600,8 @@ func startMachines(ctx context.Context, b *bigmachine.B, group *status.Group, ma
 	var wg sync.WaitGroup
 	slicemachines := make([]*sliceMachine, len(machines))
 	for i := range machines {
-		m, i := machines[i], i
+		i := i
+		m := machines[i]
```

Comment:
do you know why this is a lint error?

Reply:
We are shadowing `i`. The linter specifically ignores idiomatic short redeclaration, e.g. `i := i`, as it's an idiomatic way of shadowing/redeclaring for use in closures. See https://github.com/golang/tools/blob/dc31b401abb5f62558fefca1b34c7abab611c3c4/go/analysis/passes/shadow/shadow.go#L158-L160 .

Reply:
ok, that was basically my question, about the short redeclaration.